src/main/java/org/automation/pages/DashboardPage.java

```diff
@@ -0,0 +1,88 @@
+package org.automation.pages;
+
+import org.automation.ReturnObjects.Clickable;
+import org.automation.base.BaseTest;
+import org.checkerframework.checker.units.qual.C;
```

Comment:
Hven't seen use of this import. Can remove unused imports to keep the code clean and improve readability if not used in code.